Self-Consistent-Field/RHF.py

```diff
@@ -98,6 +98,9 @@
 Eold = 0.0
 Dold = np.zeros_like(D)
 
+E_guess = np.einsum('pq,pq->', H + H, D) + Enuc
+print('SCF Iteration %3d: Energy = %4.16f' % (0, E_guess))
```

Comment:
A bit confused here, this energy is just the one-electron energy and not really the energy of the guess (which is currently printed out in the first iteration). Psi4 uses a guess energy at iteration zero for non-vartional guesses such as SAD, those energies do include the two-electron and DFT terms.